src/data/home.json

```diff
@@ -1,4 +1,25 @@
 {
+  "header": "",
+  "description": "Description hero content",
+  "cta": {
+    "label": "Learn more",
+    "url": ""
+  },
+  "images": {
+      "small": {
+        "url": "./assets/hero-small.jpg",
+        "description": "imagen for devices small"
+      },
+      "medium": {
+        "url": "./assets/hero-medium.jpg",
+        "description": "imagen for devices medium"
+      },
+      "large": {
+        "url": "./assets/hero-large.jpg",
+        "description": "imagen for devices large"
+      }
+    },
+
```

Comment:
I think all that new data that you just added to the `home.json` file is file, but needs to be added into a new key of this json object. Just like you did for the `mainNav`.